packages/cells/src/searchprovider.ts

```diff
@@ -429,11 +429,14 @@ class MarkdownCellSearchProvider extends CellSearchProvider {
    * @param newText The replacement text.
    * @returns Whether a replace occurred.
    */
-  async replaceAllMatches(newText: string): Promise<boolean> {
+  async replaceAllMatches(
+    newText: string,
+    options?: IReplaceOptions
+  ): Promise<boolean> {
     if (this.model.getMetadata('editable') === false)
       return Promise.resolve(false);
 
-    const result = await super.replaceAllMatches(newText);
+    const result = await super.replaceAllMatches(newText, options);
```

Comment:
It looks like some changes from PR https://github.com/jupyterlab/jupyterlab/pull/16940 got included in here, let's revert them to have a clean diff :)

Reply:
Ah, I see this PR does two things. Make sense.